bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/sensor.xml

```diff
@@ -123,4 +102,121 @@
 		</channels>
 	</channel-group-type>
 
+
+	<channel-type id="sensorState">
+		<item-type>Contact</item-type>
+		<label>Contact</label>
+		<description>State of the contact: open/close</description>
+		<category>Contact</category>
+		<state pattern="%s" readOnly="true">
+			<options>
+				<option value="OPEN">Open</option>
+				<option value="CLOSED">Closed</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="sensorWakeup" advanced="true">
+		<item-type>String</item-type>
+		<label>Wakeup reason</label>
+		<description>Reasons which woke up the device: battery, button, periodic, poweron, sensor, alarm</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+
+	<channel-type id="sensorTemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Temperature from the sensor</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>CurrentTemperature</tag>
+		</tags>
+		<state readOnly="true" pattern="%f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="sensorExtTemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
```

Comment:
You should probably use the labels to differentiate the internal sensor temperature from the external sensor temperature.

Reply:
External sensors are only available for 1/1PM with the Addon installed. In this case the is no internal sensors. The Group has the label "External Sensors".
Only the sensors have a internal temp sensor. In this case it's below "Sensor".